packages/frontend/src/components/LoginForm.tsx

```diff
@@ -40,28 +40,6 @@ const CertificateChecks = {
   acceptInvalidCertificates: 'acceptInvalidCertificates',
 } as const satisfies { [P in T.EnteredCertificateChecks]: P }
 
-export function defaultCredentials(credentials?: Credentials): Credentials {
```

Comment:
why move this to the progress dialog? from what I understand it is only used for the login form

Reply:
like core does not need them to be set to null. can we remove it if it is not used?

Reply:
let's better make a new file for it